List WrkzCoin (WRKZ)

wrkzdev opened the pull request:
This is a re-submission https://github.com/bisq-network/bisq/pull/1834

- Official project URL: https://www.wrkz.work/
- Official block explorer URL: https://myexplorer.wrkz.work/

blabno commented on assets/src/main/java/bisq/asset/coins/WrkzCoin.java:
```diff
@@ -0,0 +1,28 @@
+    public WrkzCoin() {
+        super("WrkzCoin", "WRKZ", new RegexAddressValidator("^Wrkz[1-9A-Za-z^OIl]{94}"));
+    }
+}
```
Add newline at the end of file

blabno commented on assets/src/test/java/bisq/asset/coins/WrkzCoinTest.java:
```diff
@@ -0,0 +1,46 @@
+        assertInvalidAddress("WrkzXTU4REbRijuLPpds2k4BhcBGgXFpeEaXKs49D7$PFuqBYpQw2tQbAApoQLAp2iWVsoxiPmcERXhHrhtCLnzL4ezB8kAbxH");
+        assertInvalidAddress("cccd2bd37455350e7586cf9315c7f3acd3de56321aa356ff3391bd21f0bbf502");
+    }
+}
```
Add newline at the end of file

blabno reviewed (changes requested):
NACK per comments

wrkzdev committed:
List WrkzCoin (WRKZ)

Author:    wrkzdev <40448869+wrkzdev@users.noreply.github.com>
Date:      Wed Dec 19 21:44:27 2018 +0700

blabno reviewed (approved):
ACK

ripcurlx commented:
@wrkzdev Please fix the conflict and I'll merge the PR
